.github/copilot-instructions.md

```diff
@@ -1,9 +1,12 @@
 # Update Package for Latest Dependencies
 
+Instructions:
+* Always use absolute path when change directory.
```

Comment:
agent be much better on which folder to operate, when specify "use absolute path"